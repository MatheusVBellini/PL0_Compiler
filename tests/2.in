VAR y;
BEGIN
  y := 351 - 412;
END.
