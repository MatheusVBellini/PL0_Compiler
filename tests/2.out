1:10 error: Missing equal `=` symbol after this token
1:11 error: Expected a period at the end of the program

Compilation failed, 2 errors
